index.rst

```diff
@@ -1,61 +1,338 @@
+    Multiple identity providers may provide access to the same group.
+    In this case, the membership should only be withdrawn if all those identity providers stop providing the relevant information.
+    It must be possible to periodically force users to authenticate with an attribute-providing identity provider to reconfirm access to their groups, similar to IDM-0009.
+
+IDM-2002
+    Users must be able to create their own groups.
+    The owner of the group must then be able to add and remove members as they wish.
+    Owners must also be able to add additional group owners who can then also control membership in the group.
+
+IDM-2003
+    It must be possible to create groups whose membership can only be changed by Science Platform administrators.
+
+IDM-2004
+    Groups must be checked against namespace rules that, for instance, force all groups created by a user to start with a specific prefix that includes their username.
+
+IDM-2005
+    Group membership may include an expiration date.
+    When the expiration date is reached, the user will be automatically removed from the group.
+    Anyone who can control membership in the group must be able to update the expiration date.
+
+IDM-2006
+    Owners must be able to rename groups while preserving all quota grants and membership.
+
+IDM-2007
+    Owners must be able to delete groups.
+
+IDM-2008
+    Users must be able to see all of their group memberships and their expirations (if any).
+
+Quotas
+------
+
+IDM-2100
+    Groups may have one or more quota grants associated with the group.
+    These are of two types: Quotas for the group itself (such as for shared storage space), and quota that is inherited by every member of the group (granting additional personal quota).
+    These may represent file storage quotas or any other service limit that may vary by user (API rate limits, CPU equivalents for batch jobs, download size limits, or whatever may eventually be appropriate).
+    The identity management system need not understand the quotas, but it should be able to sum multiple quotas under the same label.
+
+IDM-2101
+    All members of the group must be able to view all of its quota grants.
+
+IDM-2102
+    All owners of the group must be able to request new quota grants.
+    That request should be routed to some approval process by a manager of the relevant resource, who can then grant or deny the request via the identity management web interface.
+
+IDM-2103
+    Quota grants may expire.
+    The owners of a group must be notified via email of pending quota grant expirations.
+
+Logging
+-------
+
+IDM-2200
+    All group creation, deletion, renaming, and membership changes must be logged.
+
+IDM-2201
+    Owners must be able to see all history of changes to the group via the web interface.
```

Comment:
This wording is a little ambiguous whether it means the see changes in the web interface or see changes made by the web interface.  Maybe:
"The web interface must provide owners with a history of all changes to the group."
?